test/connection-pool.test.js

```diff
@@ -1,16 +1,13 @@
 var assert = require('assert');
 var Request = require('tedious').Request;
+var fs = require('fs');
 var ConnectionPool = require('../lib/connection-pool');
 
-var connectionConfig = {
-    userName: 'test',
-    password: 'test',
-    server: 'dev1',
-    options: {
-        appName: 'pool-test',
-        database: 'test'
-    }
-};
+function getConfig() {
```

Comment:
I actually copied this logic from tedious.  I was trying to test against my sql server and needed different connection settings to test.  The main reason was that I added a port option.  I probably could have modified my hosts file and sql server to facilitate running with the existing test config.

https://github.com/pekim/tedious/blob/master/test/integration/connection-test.coffee

I wasn't sure the best way to facilitate this.